### ISSUE TITLE
Apple

### ISSUE THREAD

**Issue by hugo0830** [aws-samples/amazon-ivs-broadcast-ios-sample#55]
ITMS-90338: Non-public API usage - The app references non-public symbols in Frameworks/AmazonIVSBroadcast.framework/AmazonIVSBroadcast: _CMTimebaseCreateWithMasterClock,The app references non-public symbols in Frameworks/AmazonIVSPlayer.framework/AmazonIVSPlayer: _CMTimebaseCreateWithMasterClock. If method names in your source code match the private Apple APIs listed above, altering your method names will help prevent this app from being flagged in future submissions. In addition, note that one or more of the above APIs may be located in a static library that was included with your app. If so, they must be removed. For further information, visit the Technical Support Information at http://developer.apple.com/support/technical/

Best regards,

The App Store Team

**Comment by yuya-h-29** [comment]
I had a same issue. Let me know how to work around this stuff!

**Comment by cemnisan** [comment]
Same issue. Is there any changes?

**Comment by YaroslavBasyuk** [comment]
I have the same issue.

**Comment by rzvagelsky** [comment]
Having the same issue. Hoping for a quick resolution.

**Comment by JuliaGrasevych** [comment]
Does anybody have any updates on this? Threads on Apple Developer site are silent.

**Comment by rzvagelsky** [comment]
Updated posted on Apple dev thread: https://developer.apple.com/forums/thread/733318?page=2

**Comment by bclymer** [comment]
Hey all, as many of you probably know by now, Apple pushed an update Wednesday evening which preventing uploading new, or updating existing apps to the App Store (TestFlight included). They flagged our SDK as using a private API, `_CMTimebaseCreateWithMasterClock`. After an internal investigation, we determined that we were not using that API, but instead using `CMTimebaseCreateWithSourceClock`, which is Apple's recommended replacement API. However the definition for that function, as seen in `CoreMedia/CMSync.h`, is as follows:

```
static inline OSStatus
CMTimebaseCreateWithSourceClock(
		CFAllocatorRef CM_NULLABLE allocator,
		CMClockRef CM_NONNULL sourceClock,
		CM_RETURNS_RETAINED_PARAMETER CMTimebaseRef CM_NULLABLE * CM_NONNULL timebaseOut )
			API_AVAILABLE(macos(10.8), ios(6.0), tvos(9.0), watchos(6.0))
{
	return CMTimebaseCreateWithMasterClock(allocator, sourceClock, timebaseOut);
}
```
Which means our SDK was getting flagged as using the old, deprecated API, even though we were using the correct API. 

We notified Apple immediately notifying them of the issue. About 24 hours after we notified them they updated their [public status page](https://developer.apple.com/system-status/) informing everybody of the issue. Yesterday evening they resolved the issue and we validated our SDK (and all other apps and SDKs using the correct, non-deprecated function, were no longer blocked).

We're sorry for the inconvenience this caused, but thankful that Apple was able to resolve the issue quickly.
